Marlin/src/module/configuration_store.cpp

```diff
@@ -1895,18 +1895,18 @@ void MarlinSettings::postprocess() {
       //
       {
         #if ENABLED(BACKLASH_COMPENSATION)
-          float   *backlash_distance_mm      = backlash.backlash_distance_mm;
-          uint8_t &backlash_correction       = backlash.backlash_correction;
+          float   (&backlash_distance_mm)[XYZ] = backlash.distance_mm;
```

Comment:
This is how you make a reference that knows the size of the array it points to, so `sizeof()` will return 12.

Reply:
Oh, okay. Good to know.